[ADD] howto/website: add media page

karlosvalverde opened the pull request:
This PR adds the `media` page, links it to the `website_themes`
index and moves images from the current `pages` folder to a new one.

Task-4316611

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/12192.png)](https://mergebot.odoo.com/odoo/documentation/pull/12192)

karlosvalverde commented:
@AntoineVDV A new one for you 😉 

cc: @Cocographique

AntoineVDV reviewed (approved):
@robodoo delegate+

AntoineVDV commented:
There seems to be an issue with the image referenced on this page (see runbot). It could be that their format is not actually PNG or something like that.

bram-odoo commented:
Hey @karlosvalverde, I have two questions for you: 

1. In the "Layout" page, I have a reference to the "Company Logo", could you please add an anchor in the "Media" page?  
2. While I was looking for the anchor in your PR, I saw you wrote this brand new page with a lot of "You". Following [the content guidelines](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#writing-style), aren't we supposed to avoid that by using the imperative as much as possible?

karlosvalverde commented:
> There seems to be an issue with the image referenced on this page (see runbot). It could be that their format is not actually PNG or something like that.

It was a reference coming from the `pages.rst` file. Since I moved the pictures to a new folder (from /pages to /media), there were errors in the runbot. Now it is fixed.

> Hey @karlosvalverde, I have two questions for you:
> 
>     1. In the "Layout" page ([PR 12220](#12220)), I have a reference to the "Company Logo", could you please add an anchor in the "Media" page?
> 
>     2. While I was looking for the anchor in your PR, I saw you wrote this brand new page with a lot of "You". Following [the content guidelines](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#writing-style), aren't we supposed to avoid that by using the imperative as much as possible?

I have created anchors for that section and I have also changed the tone of voice. Actually, this style thing is quite relevant and we should be aligned with it in our next content iterations.

Thanks a lot!

karlosvalverde commented:
@robodoo r+
